specification/machinelearningservices/AzureAI.Assets/tspconfig.yaml

```diff
@@ -17,12 +17,11 @@ options:
     generate-test: true
     generate-sample: true
   "@azure-tools/typespec-ts":
-    generateMetadata: true
-    generateTest: true
-    azureSdkForJs: true
+    generate-metadata: true
+    generate-test: true
     package-dir: "ai-resources-autogen"
-    packageDetails:
-      name: "@azure/ai-resources-autogen"
+    package-details:
+      name: "@azure-rest/ai-resources-autogen"
```

Comment:
This is added by JS codegen team and considering this is RLC lib so we need to add -rest prefix.